docs/getting-started.md

```diff
@@ -44,4 +44,72 @@ Een lijst met andere ondersteunde generator opties kun je vinden in de [Generato
 Note. De prerequisite van OpenAPI Generator is JAVA. Je moet een JAVA runtime installeren voordat je OpenAPI Generator kunt gebruiken
   
 ## Probeer en test de API
-Op de Haal Centraal demo-omgeving (www.haalcentraal.nl/haalcentraal/api/) is de BRP update API nog niet beschikbaar.
+Wil je de 'BRP-Update' Web API proberen en testen? Kijk op: `https://www.haalcentraal.nl/haalcentraal/api/brpupdate`
```

Comment:
> Kijk op: `https://www.haalcentraal.nl/haalcentraal/api/brpupdate`

'Kijken' lijkt me niet de juiste term. Ik weet dat het in de Getting Started van de 2.0 beta versie van de Haal Centraal Personen Bevragen op dezelfde wijze is verwoord maar ook daar is het volgens mij niet correct. Dit verleidt nl. om deze url in een webbrowser in te geven.
Beter zou n.m.m. al zijn:

> Maak dan gebruik van `https://www.haalcentraal.nl/haalcentraal/api/brpupdate`